src/Strategy.sol

```diff
@@ -40,57 +49,81 @@ contract LiquityV2SPStrategy is BaseHealthCheck {
+    /// - Defines the lowest value that `dustThreshold` (the collateral dust threshold) can be set to
+    /// - Also reused as the fixed `ASSET_DUST_THRESHOLD`, representing the minimum asset amount
+    ///   considered worth depositing during harvests
     uint256 public constant MIN_DUST_THRESHOLD = 1e15;
 
+    /// @notice Asset dust threshold. We will not bother depositing amounts below this value on harvests
+    uint256 public constant ASSET_DUST_THRESHOLD = MIN_DUST_THRESHOLD;
+
     /// @notice Collateral reward token of the Stability Pool
     ERC20 public immutable COLL;
 
     /// @notice Liquity's price oracle for the collateral token
-    ///         Assumes the price feed is using 18 decimals
+    /// @dev Assumes the price feed is using 18 decimals
     IPriceFeed public immutable COLL_PRICE_ORACLE;
 
+    /// @notice Direct reference to the underlying Chainlink price feed used by `COLL_PRICE_ORACLE`
+    /// @dev This feed provides a read-only price for the collateral asset without
+    ///      performing Liquity's internal validity or heartbeat checks
+    AggregatorV3Interface public immutable CHAINLINK_ORACLE;
+
     /// @notice Stability Pool contract
     IStabilityPool public immutable SP;
 
     /// @notice Auction contract for selling the collateral reward token
     IAuction public immutable AUCTION;
 
-    /// @notice Factory for creating the auction contract
-    IAuctionFactory public immutable AUCTION_FACTORY;
-
     // ===============================================================
     // Constructor
     // ===============================================================
 
     /// @param _addressesRegistry Address of the AddressesRegistry
     /// @param _asset Address of the strategy's underlying asset
     /// @param _auctionFactory Address of the AuctionFactory
+    /// @param _oracle Address of the COLL/USD _read_ price oracle
     /// @param _name Name of the strategy
     constructor(
         address _addressesRegistry,
         address _asset,
         address _auctionFactory,
+        address _oracle,
         string memory _name
     ) BaseHealthCheck(_asset, _name) {
         COLL_PRICE_ORACLE = IAddressesRegistry(_addressesRegistry).priceFeed();
         (, bool _isOracleDown) = COLL_PRICE_ORACLE.fetchPrice();
         require(!_isOracleDown && COLL_PRICE_ORACLE.priceSource() == IPriceFeed.PriceSource.primary, "!oracle");
 
+        CHAINLINK_ORACLE = AggregatorV3Interface(_oracle);
+        require(CHAINLINK_ORACLE.decimals() == 8, "!decimals");
+
         SP = IAddressesRegistry(_addressesRegistry).stabilityPool();
         require(SP.boldToken() == _asset, "!sp");
         COLL = ERC20(SP.collToken());
 
-        AUCTION_FACTORY = IAuctionFactory(_auctionFactory);
-        AUCTION = AUCTION_FACTORY.createNewAuction(_asset);
+        AUCTION = IAuctionFactory(_auctionFactory).createNewAuction(_asset);
         AUCTION.enable(address(COLL));
 
+        minAuctionPriceBps = 9_000; // 90%
```

Comment:
in contradiction to one of my previous comments, we may not be better to defualt to 95% since we will be rekicking when fails

Reply:
https://github.com/yearn/yBOLD/pull/6/commits/c9531f4fc5e0e3258719d4396e5e57fa0d049748